app/grandchallenge/hanging_protocols/views.py

```diff
@@ -11,7 +11,7 @@
 
 class HangingProtocolList(LoginRequiredMixin, ListView):
     model = HangingProtocol
-    paginate_by = 50
+    paginate_by = 48

```

Comment:
Since the columns is limited to 3 per row 48 lines out nicer than 50,

Reply:
I agree. I was sticking to the value mostly used in other occurrences of `paginate_by`

Reply:
Done